client/landing/stepper/declarative-flow/design-first.ts

```diff
@@ -152,7 +153,7 @@ const designFirst: Flow = {
 						} );
 
 						if ( providedDependencies?.hasSetPreselectedTheme ) {
-							updateLaunchpadSettings( siteSlug as string, {
+							updateLaunchpadSettings( wpcom, siteSlug as string, {
```

Comment:
Worst case scenario alternative in case we can't do much: as it still feels wrong to have to pass `wpcom` to each `updateLaunchpadSettings()` call, we could have something like:

```tsx
import { updateLaunchpadSettings as updateLaunchpadSettingsReal } from '@automattic/data-stores';
import wpcom from 'calypso/lib/wp';

function updateLaunchpadSettings( siteSlug, settings ) {
	return updateLaunchpadSettingsReal( wpcom, siteSlug, settings );
}
```

or we could even create this as a helper function somewhere.

That way at least the API of `updateLaunchpadSettings` and similar wouldn't feel that awkward to the consumers.

Reply:
However, @jsnajdr's proposal in [this comment](https://github.com/Automattic/wp-calypso/pull/98417#discussion_r1919679721) seems like a much better idea.